routes/jobs/index.js

```diff
@@ -0,0 +1,28 @@
+/*jshint esversion: 6 */
+(function() {
+  'use strict';
+
+  const util = require('util');
+  const moment = require('moment');
+  const _ = require('lodash');
+  const Common = require(__dirname + '/../common');
+
+  module.exports = (app, config, ModulesClass) => {
+    
+    app.get('/jobs/:limit/:sortBy/:sortDir', (req, res) => {
```

Comment:
limit, sortBy and sortDir all should be query parameters and defaults should be set on the server side